compiler/damlc/daml-ide-core/test/Development/IDE/State/API/Testing.hs

```diff
@@ -523,17 +523,16 @@ timedSection targetDiffTime block = do
         throwError $ TimedSectionTookTooLong targetDiffTime actualDiffTime
     return value
 
-actionsToChoiceActions :: Set.Set V.Action -> [ExpectedChoiceAction]
-actionsToChoiceActions acts = Set.toList $ Set.map expectedChcAction acts
+actionsToChoiceActions :: Set.Set V.Action -> Set.Set ExpectedChoiceAction
+actionsToChoiceActions acts = Set.map expectedChcAction acts
     where expectedChcAction = \case
             V.ACreate tcon -> Create (DAP.renderPretty tcon)
             V.AExercise tcon choice -> Exercise (DAP.renderPretty tcon) (DAP.renderPretty choice)
 
 templateChoicesToProps :: V.TemplateChoices -> TemplateProp
-templateChoicesToProps tca = TemplateProp tName choicesInTpl $ Set.fromList allActions
+templateChoicesToProps tca = TemplateProp tName choicesInTpl -- $ Set.fromList allActions
```

Comment:
Looks like you forgot to remove this fully.

Reply:
:(